src/common/translation/i18n/en.json

```diff
@@ -337,13 +345,21 @@
         "text": "The Culture Kids service"
       },
       "service": {
-        "name": "Kulttuurin kummilapset -palvelu  ",
+        "name": "The Culture Kids service",
         "text": "Contact us to find out more about the service."
       },
       "godchild": {
         "name": "Culture Kids activities for children born in 2020",
         "text": "Helsinki Philharmonic Orchestra acts as the sponsor for Culture Kids born in 2020. Contact us to find out more about the Culture Kids programme for children born in 2020."
       }
     }
-  }
+  },
+  "email": {
+    "title": "Welcome to the Culture Kids programme!"
+  },
+  "email ": {
+    "heading": "Welcome to the Culture Kids programme!",
+    "text": "Welcome!"
+  },
+  " Please note! You cannot reply to this message.": ""
```

Comment:
I don't think these email texts should be here (?)

Reply:
If they are in the sheet, they will automatically also come here, we don't have a filter in the script that fetches the translations. I don't think it hurts the project to have them - but they can be moved to their "own" tab in the spreadsheet at some point.